.github/workflows/tests.yml

```diff
@@ -60,11 +60,23 @@ jobs:
       - name: Install Yarn
         run: npm install -g yarn
       # Build and install the package if it's Windows, to service shell-based tests
-      - if: ${{ matrix.os }} == "windows-latest"
+      - name: Install binary on Windows
+        if: ${{ matrix.os == 'windows-latest' }}
+        env:
+          SKIP_PRE_BUILD: true
```

Comment:
Maybe this is no longer needed?

Reply:
We're still doing a `poetry build` in here, so it will build the front end otherwise, we could of course tie this to the matrix value, so that it's picked up by any future Windows workflows (hopefully this step is removed soon though if I can figure out the `venv` stuff on the runner)

Reply:
Ah, I thought SKIP_PRE_BUILD was already set here.

Reply:
Perhaps `SKIP_PRE_BUILD: ${{ matrix.skip_pre_build }}` would make more sense?